react/features/toolbox/components/AudioRouteButton.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import {
+    findNodeHandle,
+    requireNativeComponent,
+    NativeModules,
+    Platform,
+    View
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import ToolbarButton from './ToolbarButton';
+
+import { openDialog } from '../../base/dialog';
+import { AudioRoutePickerDialog } from '../../mobile/audio-mode';
```

Comment:
The group of imports from other features should be before the group of imports from the feature itself.